packages/zk-certificates/contracts/AgeProofZkKYC.sol

```diff
@@ -3,174 +3,174 @@ pragma solidity ^0.8.0;
-    uint256 public constant timeDifferenceTolerance = 30 * 60; // the maximal difference between the onchain time and public input current time
-
-    // indices of the ZKP public input array
-    uint8 public immutable INDEX_USER_PUBKEY_AX;
-    uint8 public immutable INDEX_USER_PUBKEY_AY;
-    uint8 public immutable INDEX_IS_VALID;
-    uint8 public immutable INDEX_VERIFICATION_EXPIRATION;
-    uint8 public immutable START_INDEX_ENCRYPTED_DATA;
-    uint8 public immutable INDEX_ROOT;
-    uint8 public immutable INDEX_CURRENT_TIME;
-    uint8 public immutable INDEX_USER_ADDRESS;
-    uint8 public immutable INDEX_CURRENT_YEAR;
-    uint8 public immutable INDEX_CURRENT_MONTH;
-    uint8 public immutable INDEX_CURRENT_DAY;
-    uint8 public immutable INDEX_AGE_THRESHOLD;
-    uint8 public immutable INDEX_HUMAN_ID;
-    uint8 public immutable INDEX_DAPP_ID;
-    uint8 public immutable INDEX_PROVIDER_PUBKEY_AX;
-    uint8 public immutable INDEX_PROVIDER_PUBKEY_AY;
-    uint8 public immutable START_INDEX_INVESTIGATION_INSTITUTIONS;
-
-    constructor(
-        address _owner,
-        address _verifier,
-        address _KYCRegistry,
-        address[] memory _fraudInvestigationInstitutions
-    ) Ownable(_owner) {
-        verifier = IAgeProofZkKYCVerifier(_verifier);
-        KYCRegistry = IKYCRegistry(_KYCRegistry);
-        for (uint i = 0; i < _fraudInvestigationInstitutions.length; i++) {
-            fraudInvestigationInstitutions.push(
-                IGalacticaInstitution(_fraudInvestigationInstitutions[i])
-            );
-        }
-
-        // set public input indices according to the number of institutions
-        INDEX_HUMAN_ID = 0;
-        INDEX_USER_PUBKEY_AX = 1;
-        INDEX_USER_PUBKEY_AY = 2;
-        INDEX_IS_VALID = 3;
-        INDEX_VERIFICATION_EXPIRATION = 4;
-
-        // for each institution there are two fields containing the encrypted data of the shamir shares
-        START_INDEX_ENCRYPTED_DATA = 5;
-        uint8 institutionKeyEntries = 2 *
-            uint8(fraudInvestigationInstitutions.length);
-
-        INDEX_ROOT = 5 + institutionKeyEntries;
-        INDEX_CURRENT_TIME = 6 + institutionKeyEntries;
-        INDEX_USER_ADDRESS = 7 + institutionKeyEntries;
-        INDEX_CURRENT_YEAR = 8 + institutionKeyEntries;
-        INDEX_CURRENT_MONTH = 9 + institutionKeyEntries;
-        INDEX_CURRENT_DAY = 10 + institutionKeyEntries;
-        INDEX_AGE_THRESHOLD = 11 + institutionKeyEntries;
-        INDEX_DAPP_ID = 12 + institutionKeyEntries;
-        INDEX_PROVIDER_PUBKEY_AX = 13 + institutionKeyEntries;
-        INDEX_PROVIDER_PUBKEY_AY = 14 + institutionKeyEntries;
-
-        // The following indices are for the fraud investigation institutions and depend on the number of institutions
-        // It includes pubkeys ax and ay for each institution
-        START_INDEX_INVESTIGATION_INSTITUTIONS = 15 + institutionKeyEntries;
+  IAgeProofZkKYCVerifier public verifier;
```

Comment:
Something seems wrong with the indentation settings here. Solidity uses 4 spaces by default and in the other new Solidity files indentation is also correctly set to 4 spaces.

Reply:
yarn lint:fix solved it automatically